ci/requirements-3.6-MASTER.pip

```diff
@@ -3,3 +3,4 @@ google-auth
 google-auth-httplib2
 google-auth-oauthlib
 mock
+google-cloud-bigquery==0.26.0
```

Comment:
We might want to leave this unpinned so that the tests run on the latest version and we will know when there are breaking changes.